.github/ISSUE_TEMPLATE/epic-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Epic
+about: This template provides a basic structure for an epic issues.
+title: '[Epic] <title>'
+labels: Epic
+---
+
+### Goal
+
+_Introduction_
+
+### Success Metrics
+
+### Work items
```

Comment:
NIT: prefer capitalizing all words in headers, here and elsewhere. (Or at least consistently capitalizing only the first word)